float32 motor1
float32 motor2
float32 motor3
float32 motor4